projects/admin/src/app/record/operation-logs/operation-logs.component.html

```diff
@@ -58,6 +58,9 @@ <h5 class="modal-title" translate>Operation history</h5>
       {{ metadata.date | dateTranslate :'shortDate' }} - {{ metadata.date | dateTranslate :'mediumTime' }}
     </div>
     <div class="col-2 operation">{{ metadata.operation | translate }}</div>
-    <div class="col">{{ metadata.user_name }}</div>
+    <div class="col">
+      {{ metadata.user_name }}
+      <ng-container *ngIf="metadata.library">- {{ metadata.library.name }}</ng-container>
```

Comment:
COSMETIC: Is there a missing space before the hyphen ? 
`- {{ metadata.library.name }}` or ` - {{ metadata.library.name }}`